### ISSUE TITLE
[webview_flutter] Keyboard suggestions can be lost

### ISSUE THREAD

**Issue by mklim** [flutter/flutter#37989]
To reproduce:

1. Run `webview_flutter/example` `v0.3.11+2` on an Android device using a soft keyboard that normally provides suggestions based on what's been previously typed in a text field.
2. Tap into a text field. Observe that the soft keyboard is shown as expected.
3. Enter text. Observe text is correctly entered.
4. Minimize keyboard using the Android back button.
5. Tap back into the same text field. Observe that keyboard is shown again as expected.

**Expected**: Keyboard should display suggestions related to the text field.
**Actual**: Keyboard does not display anything related to the current text.

This is related to flutter/plugins#1965. InputEvents are being incorrectly handled by WebView after WebView is resized. This could be an underlying issue in Chromium. We should try to reproduce with a non-Flutter app and file a Chromium bug if possible.

**Comment by hisaichi5518** [comment]
By doing the following, it no longer reproduces in my project.

1. Open android/app/src/main/AndroidManifest.xml
2. Change the "adjustSize" of "android:windowSoftInputMode" to "adjustPan"
3. build

```diff
diff --git a/android/app/src/main/AndroidManifest.xml b/android/app/src/main/AndroidManifest.xml
index ab0b9ab..3312c03 100644
--- a/android/app/src/main/AndroidManifest.xml
+++ b/android/app/src/main/AndroidManifest.xml
@@ -18,7 +18,7 @@
             android:theme="@style/LaunchTheme"
             android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|layoutDirection|fontScale|screenLayout|den
             android:hardwareAccelerated="true"
-            android:windowSoftInputMode="adjustResize">
+            android:windowSoftInputMode="adjustPan">
             <!-- This keeps the window background of the activity showing
                  until Flutter renders its first frame. It can be removed if
                  there is no splash screen (such as the default splash screen
```

### Cons

- All Flutter screens are affected by this change
- Cannot scroll to the bottom when the keyboard is displayed

### See also

https://developer.android.com/guide/topics/manifest/activity-element.html#wsoft

**Comment by TahaTesser** [comment]
Hi @mklim 
Is this issue still valid?
Thank you

**Comment by mklim** [comment]
@TahaTesser yes, unless there's been a fix for this I've missed that hasn't been linked to this bug.

**Comment by darshankawar** [comment]
This issue is reproducible on latest `dev channel (1.20.0-0.0.pre)` and with latest version of `webview_flutter`.

<details>
<summary>flutter doctor -v</summary>

```
[✓] Flutter (Channel dev, 1.20.0-0.0.pre, on Mac OS X 10.15.2 19C57, locale en-US)
    • Flutter version 1.20.0-0.0.pre at /Users/deeptibelsare/Documents/Fluttersdk/flutter
    • Framework revision d9653445f4 (8 days ago), 2020-06-09 18:43:03 -0400
    • Engine revision e8c13aa012
    • Dart version 2.9.0 (build 2.9.0-14.0.dev 5c1376615e)

 
[!] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/deeptibelsare/Library/Android/sdk
    • Platform android-29, build-tools 29.0.3
    • ANDROID_HOME = /Users/deeptibelsare/Library/Android/sdk
    • ANDROID_SDK_ROOT = /Users/deeptibelsare/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    ✗ Android license status unknown.
      Try re-installing or updating your Android SDK Manager.
      See https://developer.android.com/studio/#downloads or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions.

 
[✓] Xcode - develop for iOS and macOS (Xcode 11.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3, Build version 11C29
    • CocoaPods version 1.9.3

[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 45.1.1
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[✓] Connected device (1 available)
    • Android SDK built for x86 • emulator-5554 • android-x86 • Android 8.1.0 (API 27) (emulator)

! Doctor found issues in 1 category.

```

</details>

**Comment by blasten** [comment]
This has been fixed in Flutter 1.22/ WebView 1.0 - use hybrid composition in the Flutter WebView plugin.

https://github.com/flutter/plugins/tree/master/packages/webview_flutter#using-hybrid-composition
